editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -1386,10 +1386,10 @@ function forciblyUpdateDataUID(
   originatingElement: TS.Node,
   props: JSXAttributes,
   existingHighlightBounds: Readonly<HighlightBoundsForUids>,
-  alreadyExistingUIDs: ReadonlyArray<string>,
+  alreadyExistingUIDs: Set<string>,
 ): WithParserMetadata<JSXAttributes> {
-  const keysToAvoid = [...alreadyExistingUIDs, ...Object.keys(existingHighlightBounds)]
-  const uid = generateUID(keysToAvoid)
+  const uid = generateUID(alreadyExistingUIDs)
```

Comment:
`existingHighlightBounds` is not included here, why is it not needed anymore?

Reply:
It was used previously to try and track uids that had been generated mid-parsing while not doing anything mutable. But that can be completely superseded by using the mutable `Set`.